README.md

```diff
@@ -23,13 +23,12 @@
 ### Dependencies
 ```
 git clone https://github.com/iojs/website.git
-npm install -g gulp
 npm install
 ```
 
 ### Local Development
 ```
-gulp
+npm run build
```

Comment:
@Fishrock123 The local `gulp` binary is added to `PATH` in `npm` scripts. Leaving this line as `gulp` would require one to globally install `gulp` (which this PR tries to avoid by using the local `gulp` binary through the `npm` script instead).